kongswap_adaptor/src/withdraw/withdraw_retry.rs

```diff
@@ -425,29 +309,25 @@ async fn test_withdraw_retry() {
         allowance_1.owner_account,
     );
 
-    {
-        // Check the correctness of the balances after deposit
-        let asset_0_balance = make_default_balance_book()
-            .fee_collector(2 * FEE_SNS)
-            .external_custodian(amount_0_decimals - 2 * FEE_SNS);
-
-        let asset_1_balance = make_default_balance_book()
-            .fee_collector(2 * FEE_ICP)
-            .external_custodian(amount_1_decimals - 2 * FEE_ICP);
-
-        // This should now work without panicking
-        let result_deposit = kong_adaptor.deposit(DepositRequest { allowances }).await;
-
-        let balances = Balances {
-            timestamp_ns: 0,
-            asset_to_balances: Some(btreemap! {
-                asset_0.clone() => asset_0_balance,
-                asset_1.clone() => asset_1_balance,
-            }),
-        };
-
-        assert_eq!(result_deposit, Ok(balances));
-    }
+    // We overwrite the balances to simulate the case after a
+    // happy deposit.
```

Comment:
This comment is helpful already, but it isn't 100% focused on the essence of this test. 

I think the ideal comment would remind the reader that this test is about checking that after a failed withdrawal, there can be a retry, and if that succeeds, the owner finally gets back the same as they expect from a normal withdraw.